org.eclipse.lsp4xml/src/main/java/org/eclipse/lsp4xml/extensions/contentmodel/participants/ContentModelHoverParticipant.java

```diff
@@ -54,6 +55,10 @@ public Hover onAttributeName(IHoverRequest hoverRequest) throws Exception {
 		try {
 			ContentModelManager contentModelManager = hoverRequest.getComponent(ContentModelManager.class);
 			DOMAttr attribute = (DOMAttr) hoverRequest.getNode();
+			Hover temp = XSISchemaModel.computeHoverResponse(attribute, hoverRequest);

```

Comment:
need sa comment explaining why it's not handled like other hovers through contentModelManager. findCMElement